### PR TITLE
chore(CHANGELOG): update to v1.0.0-alpha.5

### PR DESCRIPTION
**Pull request by pinussilvestrus**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,18 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
...
+
+* `FEAT`: validate cloud element templates against JSON Schema ([#561](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/561))
+* `FEAT`: use components instead of elements for all entries ([#590](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/590))
+* `FIX`: set correct title to element templates select ([#591](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/591))
+* `DEPS`: update to `@bpmn-io/properties-panel@0.11.0`
+* `DEPS`: update to `@bpmn-io/element-templates-validator@0.5.0`
+
+### BREAKING CHANGES
+
+* `component` property of an entry must be an actual component, not an element
```
Comment (barmac) [inline CHANGELOG.md:19]: Let's link some guidance here. I think the original PR could be sufficient.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,18 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
...
 
+## 1.0.0-alpha.5
+
+* `FEAT`: validate cloud element templates against JSON Schema ([#561](https://github.com/bpmn-io/bpmn-js-properties-panel/issues/561))
+* `FEAT`: use components instead of elements for all entries ([#590](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/590))
+* `FIX`: set correct title to element templates select ([#591](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/591))
+* `DEPS`: update to `@bpmn-io/properties-panel@0.11.0`
+* `DEPS`: update to `@bpmn-io/element-templates-validator@0.5.0`
+
+### BREAKING CHANGES
```
Comment (barmac) [inline CHANGELOG.md:17]:
```suggestion
### Breaking Changes
```

File: CHANGELOG.md (hunk 1 above)
Comment (pinussilvestrus) [reply CHANGELOG.md]: 👍  I added it accordingly.

File: CHANGELOG.md (hunk 2 above)
Comment (pinussilvestrus) [reply CHANGELOG.md]: Good catch, I just noticed we were quite inconsistent in the past. I updated it via https://github.com/bpmn-io/bpmn-js-properties-panel/pull/595/commits/2c4c677348a52e141e54c829383bb5008f73de64

**Review by barmac — APPROVED** [review]

**Pull request merged**